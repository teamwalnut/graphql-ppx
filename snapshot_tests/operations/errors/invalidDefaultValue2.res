/**
 * Should throw with type \"InputObject\"
 */
module Case1 = %graphql(`
  query invalidDefaultValues {
    invalidDefaultValues(
        int: {test: "xxx"}
    )
  }
`)
