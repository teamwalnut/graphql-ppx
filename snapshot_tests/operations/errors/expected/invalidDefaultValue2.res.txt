
  [1;31mWe've found a bug for you![0m
  [36msnapshot_tests/operations/errors/invalidDefaultValue2.res[0m:[2m7:14-26[0m

  5 [2mâ”‚[0m   query invalidDefaultValues {
  6 [2mâ”‚[0m     invalidDefaultValues(
  [1;31m7[0m [2mâ”‚[0m         int: [1;31m{test: "xxx"}[0m
  8 [2mâ”‚[0m     )
  9 [2mâ”‚[0m   }
 10 [2mâ”‚[0m `)

  Invalid argument. Expected "Int" but received "InputObject!".

